.circleci/config.yml

```diff
@@ -1,90 +1,114 @@
 version: 2.0
 
 aliases:
-  - &docker_agent datadog/docker-dd-agent
-  - agent_env: &agent_env
-    - DD_APM_ENABLED=true
-    - DD_BIND_HOST=0.0.0.0
-    - DD_API_KEY=invalid_key_but_this_is_fine
-  - &load_composer_deps
+  - &CACHE_COMPOSER_KEY
     key: 'betav1-composer-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "composer.json" }}'
-  - &composer_install_deps
-    name: Composer installing dependencies
-    command: composer install -n
-  - &save_composer_deps
-    key: 'betav1-composer-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "composer.json" }}'
-    paths:
-      - vendor/
-  - &test_results_path
-    path: test-results
-  - &php_test_defaults
+
+  - &CACHE_NPM_KEY
```

Comment:
Using upper-case names for anchors with consistent prefixes to make it easier to identify what they are for.